jenkins-slaves/README.md

```diff
@@ -17,7 +17,7 @@ the ODS [jenkins shared library](https://github.com/opendevstack/ods-jenkins-sha
 1. [Scala & SBT](scala/README.md)
 1. [Python](python/README.md)
 
-If you create a new slave please add a `README.md` inside its directory, based on [this template](../__JENKINS_SLAVE_TEMPLATE_README.md). 
+If you create a new slave please add a `README.md` inside its directory, based on [this template](../docs/modules/ROOT/pages/__JENKINS_SLAVE_TEMPLATE_README.adoc). 
```

Comment:
Not sure that we should have this at all in the Markdown file? Is there no adoc file with the content you edited?

Reply:
Yes, I would also prefer an Markdown file instead of an adoc, but this discussion is out of topic.
All I want with this PR is to make sure that the link to the template works again.